external-crates/move/crates/move-vm-profiler/src/lib.rs

```diff
@@ -356,9 +356,7 @@ macro_rules! profile_dump_file {
 #[macro_export]
 macro_rules! gas_profiler_feature_enabled {
     ($($tt:tt)*) => {
```

Comment:
What are these arguments supposed to be btw?

Reply:
It's a type tree, i.e. anything you want just gets put through as-is.